crates/holochain_zome_types/src/block.rs

```diff
@@ -73,6 +74,7 @@ type IpV4 = std::net::Ipv4Addr;
 pub enum BlockTarget {
     /// Some cell did bad at the happ level.
     Cell(CellId, CellBlockReason),
+    NodeDna(kitsune_p2p_block::NodeId, DnaHash, NodeSpaceBlockReason),
```

Comment:
In any case this line could use comments like the others.

Reply:
@maackle earlier designs of this system worked like that, and were abandoned because most of the emergent states are nonsense, so i setup the enum to mirror our actual infra, which has a natural sense of specificity from nodes/connections, to spaces, to cells

this was designed pairing with @neonphog 

e.g. an ip+agent doesn't mean anything (the agent is unblocked if they're on a different ip?), blocking a dna also doesn't mean anything (effectively block yourself?)